src/mon/MonCommands.h

```diff
@@ -790,9 +790,6 @@ COMMAND("osd test-reweight-by-pg " \
 	"name=pools,type=CephPoolname,n=N,req=false",			\
 	"dry run of reweight OSDs by PG distribution [overload-percentage-for-consideration, default 120]", \
 	"osd", "rw", "cli,rest")
-COMMAND("osd thrash " \
-	"name=num_epochs,type=CephInt,range=0", \
-	"thrash OSDs for <num_epochs>", "osd", "rw", "cli,rest")
 COMMAND("osd df " \
```

Comment:
need to remove the test of `test_ceph_argparse.TestOSD.test_thrash ` in `src/test/pybind/test_ceph_argparse.py`. it uses `get_command_descriptions` to get the command descriptions. and `get_command_description` includes `mon/MonCommands.h` for the list of available mon commands.